electron/src/window/WindowManager.ts

```diff
@@ -58,15 +57,9 @@ export const sendActionToPrimaryWindow = (channel: string, ...args: any[]): void
   const primaryWindow = getPrimaryWindow();
 
   if (primaryWindow) {
-    logger.info(`Sending action "${channel}" to window with ID "${primaryWindow.id}":`, ...args);
-    const focusedWebContents = WebViewFocus.getFocusedWebContents();
-    if (focusedWebContents) {
```

Comment:
We actually want the primary window's web contents, so reverting [part of #3197](https://github.com/wireapp/wire-desktop/pull/3197/files#diff-9c051b78156885d99242e715177716c0R63).